app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -1104,65 +1118,46 @@ public void displayAllMarkers() {
      * @param filterForAllNoneType true if we filter places with all none button
      */
     @Override
-    public void filterMarkersByLabels(List<Label> selectedLabels, boolean displayExists,
-                                      boolean displayNeedsPhoto,
-                                      boolean filterForPlaceState,
-                                      boolean filterForAllNoneType) {
-        if (selectedLabels.size() == 0 && filterForPlaceState) { // If nothing is selected, display all
```

Comment:
Why this if is removed @VitalyVPinchuk ?

Reply:
Hello @neslihanturan 
This wasn't removed. I just made a little bit of refactoring to make it shorter and clearer.
Originally, there was `if else` statement, both parts of which were exactly the same except for nested `for` loop to implement type filter by comparing selected types to place's label.
So I merged them into a single block of code and replaced nested `for` loop with `selectedLabels.contains(place.getLabel())` which is called when filtering needed. This should not affect performance.

Reply:
Thank you @VitalyVPinchuk , next time please send another PR for style fixes or other issue fixes. Otherwise it makes harder to review for us. For now it is okay.

Reply:
I see. Sorry.